spec/models/index_queue_spec.rb

```diff
@@ -17,6 +17,18 @@
         expect(subject.depth).to be_nil
       end
     end
+    describe 'when response is nil' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.